docs/topics/whatsnew-eap.md

```diff
@@ -80,6 +80,49 @@ These features are available in preview, and we encourage you to try them and sh
 
 [See the full list of Kotlin language design features and proposals](kotlin-language-features-and-proposals.md).
 
+### Non-local break and continue
+
+Kotlin %kotlinEapVersion% adds a preview of another long-awaited feature, an opportunity to use non-local `break` and
+`continue`. This feature expands the toolset you can use in the scope of inline functions and reduces boilerplate code
+in your project.
+
+Previously, you could only use non-local returns. Now, Kotlin also supports `break` and `continue` [jump expressions](returns.md)
```

Comment:
Maybe the "before/after" will show better the profit of using non-local break and continue? 
What do you think?

Reply:
We can try, but I'm afraid the most important part with the run lambda get's cut. Please check. we can also do two separate blocks one below the other